helm/README.md

```diff
@@ -0,0 +1,41 @@
+# Oddities with MAS
+
+## general
+
+After initialising the pod you need to sync MAS with it's config
+
+```bash
+kubectl exec -it -n mas deployments/mas -- mas-cli config sync
```

Comment:
the sync could be run from an init container. since the migrations and basic sync are now performed automatically on start up we added:
```
    - name: mas-sync-clients
      image: ghcr.io/matrix-org/matrix-authentication-service:0.9.0
      command: ["/usr/local/bin/mas-cli", "config", "sync", "--config=/config/config.yaml", "--prune"]
      volumeMounts:
        - mountPath: /config
          name: config
```